docs/content/usage/scenarios/_index.md

```diff
@@ -20,8 +20,8 @@ Choosing the "Custom" scenario will allow you to fine-tune your simulation and a
 
 ![Choose scenario](/images/usage/scenarios/choose-scenario.png "Choose a scenario")
 
-To exit a scenario and select another one, click on "Start Over".
-
+To exit a scenario and select another one, click on "Reset".
+<!--- TODO fix the screenshot --->
```

Comment:
Can we just fix it instead of leaving a TODO?

Reply:
Yes, I left this here because I knew I'll forget this :D